app/api/users.py

```
@@ -0,0 +1,200 @@
+) -> UserResponse:
+    """Get user by ID."""
+    from sqlalchemy import select
+    from uuid import UUID
+    
+    try:
+        user_uuid = UUID(user_id)
+    except ValueError:
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail="Invalid user ID format"
+        )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)